src/main/scala/org/globalforestwatch/features/FeatureRDDFactory.scala

```diff
@@ -1,9 +1,20 @@
 package org.globalforestwatch.features
 
+import java.util.HashSet
+
 import cats.data.NonEmptyList
+import com.vividsolutions.jts.geom.{Geometry, Point, Polygon}
 import geotrellis.vector
+import geotrellis.vector.io.wkb.WKB
```

Comment:
Codacy found an issue: [Use brackets to group imports from the same package](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5902460)